src/ansys/sherlock/core/errors.py

```diff
@@ -1124,3 +1124,20 @@ def str_itr(self):
 
         assert self.error_array is None
         return [f"Export test points error: {self.message}"]
+
+
+class SherlockExportAllTestFixtures(Exception):
+    """Contains the errors raised when test fixtures cannot be exported."""
+
+    def __init__(self, message=None, error_array=None):
+        """Initialize error message."""
+        self.message = message
+        self.error_array = error_array
+
+    def str_itr(self):
```

Comment:
ExportAllICTTestFixtures RPC does returns a return code only. It does not return a list of any errors or anything. So str_itr() method is not necessary here.